scripts/gen_gdt.py

```diff
@@ -8,9 +8,14 @@
 import sys
 import struct
 import os
+import elftools
 from elftools.elf.elffile import ELFFile
 from elftools.elf.sections import SymbolTableSection
 
+if float(elftools.__version__) < 0.24:
```

Comment:
This will fail if elftools `0.24.1` is released, or if the locale uses `,` for the decimal separator (Portuguese, for instance).  Better to use `LooseVersion` from `distutils.version` instead:

    from distutils.version import LooseVersion

    if LooseVersion(elftools.__version__) < LooseVersion('0.24'):
         boom()